examples/string/protein-links-detailed.py

```diff
@@ -22,4 +24,4 @@
         relation='IAO:0002436'
     )
 
-    koza.write(source_name, [protein_a, protein_b, pairwise_gene_to_gene_interaction])
+    collect(source_name, protein_a, protein_b, pairwise_gene_to_gene_interaction)
```

Comment:
we should probably go with write long term? more explicit than collect. I know i'm the one who originally called this collect - just noting here

Reply:
Sounds good to me, rather than doing it here, I made issue #21